debian/control

```diff
@@ -58,7 +58,7 @@ Build-Depends: automake,
                libnuma-dev <pkg.ceph.crimson>,
                libpciaccess-dev <pkg.ceph.crimson>,
                libsctp-dev <pkg.ceph.crimson>,
-               libsnappy-dev,
+               libsnappy-dev (>= 1.1.9),
```

Comment:
ubuntu focal only has 1.1.8. so i'd prefer keeping as it is. see https://packages.ubuntu.com/focal/libsnappy-dev

Reply:
Yes, of course, keep it as it is for now.

The idea here is that, at some point, folks will try and fail to compile Ceph on OSes that have snappy 1.1.9.

Reply:
Ah, nevermind - I'm reading your posts in the wrong order ;-)